### ISSUE TITLE
[Bug] Flashlight.TurnOnAsync / Off crash on iPad 5th gen (no flash on those)

### ISSUE THREAD

**Issue by softlion** [xamarin/Essentials#1834]
### Description
On iPad without flash lights, using Flashlight.TurnOnAsync / TurnOffAsync is crashing.

https://en.wikipedia.org/wiki/IPad_(5th_generation)

### Expected Behavior
Instead it should does nothing.  
Or provide a method Flashlight.IsSupported()

### Actual Behavior
Crash 

![image](https://user-images.githubusercontent.com/190756/122516988-bf72f100-d00f-11eb-9070-b3cedc2f3ae6.png)

### Basic Information

- Version with issue: all
- Last known good version: none
- IDE: VS19
- Platform Target Frameworks: 
  - iOS:  all
- Affected Devices: iPad without flash lights

**Comment by jamesmontemagno** [comment]
Yes, we throw an exception when it is not supported -> https://github.com/xamarin/Essentials/blob/main/Xamarin.Essentials/Flashlight/Flashlight.ios.cs#L26
